configs/gb200-fp4-mtp-setup.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+BRANCH="baizhou/gb200-spec"
+
+cd /sgl-workspace/sglang
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)